### PR TITLE
Adjust ADR based on new approach to domain data

### PR DESCRIPTION
**Pull request by JonathanHallam**
https://github.com/alphagov/content-publisher/pull/1658 marks a change in how we approach storing copy for supertypes. This updates the relevant ADR accordingly

### CODE REVIEW

File: docs/adr/0004-editing-microcopy.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ Although we could use translations to extract all of the strings in the applicat
...
    * **Big guidance** is extracted into it's own Markdown files and stored alongside the corresponding HTML page that shows it. For example, the guidance for creating a new document is stored in `app/views/new_document/guidance.md`.
-   * **Domain data** that's static is extracted into custom YAML files. This application has two static models (DocumentTypeSchema and SupertypeSchema) that encapsulate domain concepts where the data is part of the application. We think domain data - whether it's a backend setting or a label - should be defined in one place so it can change together. In this application, we've chosen to store the backing files for the two static models in `app/formats`.
+   * **Domain data** that's static is stored in a number of custom YAML files. This application has two static models (DocumentTypeSchema and SupertypeSchema) that encapsulate domain concepts where the data is part of the application. We have split up domain data based on whether it's used in a backend setting or as a label for use in the frontend'. The former is stored in `app/formats` whilst the latter is in `config/locales/en`.
```
Comment (benthorner) [inline docs/adr/0004-editing-microcopy.md]: Almost! This ADR was out-of-date already: we don't store the config in `app/formats` anymore. Maybe just lose that bit?

Comment (JonathanHallam) [reply docs/adr/0004-editing-microcopy.md]: oops should have spotted this, sorry! Will just remove the last sentence

File: docs/adr/0004-editing-microcopy.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ Although we could use translations to extract all of the strings in the applicat
...
    * **Big guidance** is extracted into it's own Markdown files and stored alongside the corresponding HTML page that shows it. For example, the guidance for creating a new document is stored in `app/views/new_document/guidance.md`.
-   * **Domain data** that's static is extracted into custom YAML files. This application has two static models (DocumentTypeSchema and SupertypeSchema) that encapsulate domain concepts where the data is part of the application. We think domain data - whether it's a backend setting or a label - should be defined in one place so it can change together. In this application, we've chosen to store the backing files for the two static models in `app/formats`.
+   * **Domain data** that's static is stored in a number of custom YAML files. This application has two static models (DocumentTypeSchema and SupertypeSchema) that encapsulate domain concepts where the data is part of the application. We have split up domain data based on whether it's used in a backend setting or as a label for use in the frontend'.
```
Comment (benthorner) [inline docs/adr/0004-editing-microcopy.md]:
```suggestion
   * **Domain data** that's static is stored in a number of custom YAML files. This application has two static models (DocumentTypeSchema and SupertypeSchema) that encapsulate domain concepts where the data is part of the application. We have split up domain data based on whether it's used in a backend setting or as a string for use in the frontend. The latter are extracted at the top-level of the translation hierarchy..
```

**Comment by kevindew** [comment]
@JonathanHallam Thanks for revising this ADR - could you also add the date that was revised to go alongside the date the initial decision was made?

As ADR's represent a decision at a point in time, and not the current status of the app, it's good to capture the context of any later revisions to the decision.

**Pull request merged**